README.md

```
@@ -51,73 +64,61 @@
-   # Homebrew installation
-   /opt/homebrew/bin/node /path/to/vercel-mcp/dist/index.js
-   
-   # NVM installation
-   ~/.nvm/versions/node/v18.x.x/bin/node /path/to/vercel-mcp/dist/index.js
+2. **Get your Vercel Access Token**:
+   - Go to https://vercel.com/account/tokens 
+   - Create a new token with appropriate permissions
+   - Use it in your Docker run command or docker-compose.yml
+
+3. **Docker Compose (Recommended)**:
+   ```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)